docs/getting-started/installation.rst

```diff
@@ -72,9 +72,9 @@ The following are a list of optional dependencies and their purpose:
 +-----------------------------------+-----------------------------------------+
 | ``tqdm``                          | Status bars for monitoring filters      |
 +-----------------------------------+-----------------------------------------+
-| | ``trimesh``                     | |                                       |
-| | ``rtree``                       | | Vectorised ray tracing           |
-| | ``pyembree``                    |                                         |
+| ``trimesh``                       |                                         |
```

Comment:
I had added the extra pipes here to ensure that the dependencies each formatted on their own line; the result of my googling how to do line breaks in an rst table. In my testing having it the way you've change it to meant they all appeared on one line together which I thought looked bad. I'm not wedded to my way though, so you do what you think is best.

Reply:
I was getting errors indicating the table was incorrectly formatted.  You can build the docs locally to check this.  I'm going to keep it the way it is.